modules/nf-core/modules/deeparg/downloaddata/main.nf

```diff
@@ -7,13 +7,22 @@ process DEEPARG_DOWNLOADDATA {
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
         'https://depot.galaxyproject.org/singularity/deeparg:1.0.2--pyhdfd78af_1' :
         'quay.io/biocontainers/deeparg:1.0.2--pyhdfd78af_1' }"
+    /*
+    We have to force singularity to run with -B to allow reading of a problematic file with borked read-write permissions in an upstream dependency (theanos).
+    Original report: https://github.com/nf-core/funcscan/issues/23
+    */
+    containerOptions { "${workflow.containerEngine}" == 'singularity' ? '-B $(which bash):/usr/local/lib/python2.7/site-packages/Theano-0.8.2-py2.7.egg-info/PKG-INFO' : '' }
```

Comment:
I'm wondering if there can be a situation on Windows where you have bash available (as required by nextflow) but not `which`. Probably not. I guess we can handle it later anyway.

Would suggest to simply turn it into string interpolation.

```suggestion
    containerOptions "${workflow.containerEngine == 'singularity' ? '-B $(which bash):/usr/local/lib/python2.7/site-packages/Theano-0.8.2-py2.7.egg-info/PKG-INFO' : '' }"
```